### PR TITLE
Fix/master/use maven central

### PR DESCRIPTION
**Pull request by anestos**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,7 +2,8 @@ apply plugin: 'java'
 apply plugin: 'maven'
 
 group = 'io.curity.identityserver.plugin'
-version = '2.2.2'
+version = '1.0.0'
```
Comment (travisspencer) [inline build.gradle:5]: could cause confusion to go down in version numbers, so I'd suggest to not.

**Review by travisspencer — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (anestos) [reply build.gradle:5]: We haven't published a release on this plugin though

Comment (anestos) [reply build.gradle:5]: What version should i put ?

Comment (travisspencer) [reply build.gradle:5]: ah, OK. Then, 1.0.0 is OK I suppose. Otherwise, I was thinking 2.3.0

**Pull request merged**